build.sbt

```diff
@@ -255,11 +259,12 @@ lazy val ctl = project
   .enablePlugins(AutomateHeaderPlugin)
   .settings(commonSettings)
   .settings (
-    resolvers += "bmjames Bintray Repo" at "https://dl.bintray.com/bmjames/maven",
+    resolvers += Resolver.bintrayRepo("bkirwi", "maven"),
     libraryDependencies ++= Seq(
-      "org.scalaz"  %% "scalaz-core"   % scalazVersion,
-      "org.scalaz"  %% "scalaz-effect" % scalazVersion,
-      "net.bmjames" %% "scala-optparse-applicative" % "0.5"
+      "org.typelevel"           %% "cats"        % catsVersion,
+      "org.typelevel"           %% "cats-effect" % catsEffectVersion,
+      "com.monovore"            %% "decline"     % declineVersion,
+      "com.github.benhutchison" %% "mouse"       % mouseVersion
```

Comment:
How much can `mouse` supplant the variety of methods we use in scalaz. e.g. `option` or the `? a | b` type of expressions?

Is it expected that all of them will be made available via `mouse`?